underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -73,7 +73,7 @@ public HdfsUnderFileSystem(String fsDefaultName, TachyonConf tachyonConf, Object
     try {
       mFs = path.getFileSystem(tConf);
     } catch (IOException e) {
-      LOG.error("Exception thrown when trying to get FileSystem for " + mUfsPrefix, e);
```

Comment:
If the format string specifies `n` parameters and the arguments are `n` parameters followed by an exception, the exception will be printed with stack trace on the line following the log message.